src/wp-includes/blocks/image.php

```diff
@@ -16,11 +16,13 @@
  * @return string The block content with the data-id attribute added.
  */
 function render_block_core_image( $attributes, $content, $block ) {
+	if ( false === stripos( $content, '<img' ) ) {
```

Comment:
```suggestion
	if ( ! str_contains( $content, '<img' ) ) {
```

Reply:
FWIW, [this code](https://github.com/WordPress/gutenberg/blob/0e2e55499f2e4bc5afafcd74040cd9074283f85d/packages/block-library/src/image/index.php#L19) comes in via npm package sync from Gutenberg, so we'll need to fix it there.

There's another package sync planned for Monday; would it be okay to leave as-is until then if we prepare the change in GB and line it up for that package sync?

Reply:
Prepped a small PR at https://github.com/WordPress/gutenberg/pull/55302, which can be included in the next package sync.

Reply:
(Also noting that strictly speaking, the code isn't 100% equivalent, since `stripos` is case-insensitive, whereas `str_contains` isn't. While block editor-generated code shouldn't contain non-lowercase HTML tags, we'd need to wrap `$content` in `strtolower()` to be 100% equivalent.)

Reply:
for follow-up to complete the circle: this and the suggestion above it replacing `stripos()` with `str_contains()` introduce defects into the code.

thank you everyone who noticed.